docs/standard/microservices-architecture/index.md

```diff
@@ -3,7 +3,7 @@ title: .NET Microservices. Architecture for Containerized .NET Applications
 description: .NET Microservices Architecture for Containerized .NET Applications | Front Matter
```

Comment:
it would be good to start adding real descriptions to these topics. This is what is displayed on search results and should be a call to action for folks to visit your link.

![image](https://user-images.githubusercontent.com/12971179/40099204-e88f8c8e-5893-11e8-811c-214737c280f8.png)

Reply:
👍